### ISSUE TITLE
loss nan

### ISSUE THREAD

**Issue by feiyang2010jin** [Stick-To/CenterNet-tensorflow#14]
hello   I trained in my dataset  and also load pre train weight that you upload, but  still get loss is nan,
can you give me som advice ?  thanks

**Comment by Stick-To** [comment]
the pretrained weight I uploaded only had pretrained several epochs, may more precise pretrained weight could help. But I can't provide, Sorry

And  map adjusting  learning rate also help
